AzureADApplicationRegistration/functions/Set-AzureADApplicationReplyUrls.ps1

```diff
@@ -27,15 +31,20 @@ function Set-AzureADApplicationReplyUrls {
     [array]$AADAppReplyUrls = $AADAppReplyUrlsNoSpace.Split(",")
 
     # Get amd existing app
-    $ExistingAADApp = Get-AzureADApplication -SearchString $AADAppName
+    $ExistingAADApp = Get-AzureADApplication -SearchString $AADAppName | where DisplayName -EQ $AADAppName
+
+
+    if ($null -eq $ExistingAADApp) {
```

Comment:
I guess this is a powershell kind of writing pattern but it reads sooo yoda to me :) `if null the value is`

Reply:
I'm just checking if `$ExistingAADApp` has a value if not throw an error. 

And yes, it's a Powershell thing or more precise PSScriptAnalyzer rule. `$null` has to be on the left side when using comparison with collections.